src/patterns/components/offline-notification/overview.hbs

```diff
@@ -0,0 +1,7 @@
+---
+order: 10
+notes: |
+  In page notification about lack of internet connectivity.
+---
+
+{{{embed "patterns.components.offline-notification.example"}}}
```

Comment:
Makes sense. I reorganized things. I also renamed this whole pattern to message, with the offline message as an example of a warning message, since that's really more accurate.